plugins/wallet_api_plugin/wallet_api_plugin.cpp

```diff
@@ -25,7 +25,7 @@ using namespace eosio;
    [&api_handle](string, string body, url_response_callback cb) mutable { \
           try { \
              INVOKE \
-             cb(http_response_code, fc::time_point::maximum(), fc::variant(result)); \
+             cb(http_response_code, fc::time_point::now() + fc::days(365), fc::variant(result)); \
```

Comment:
Original 30ms can be too short, but 365 days might be too long. Do we want to set it to some reasonable number like 10 seconds so feedback can be returned in case keosd is really down?

Reply:
Got it.